README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
+              "python"
+            ],
+            "suggests": [
+              "libgnome-keyring0"
+            ]
+          },
+          "redhat_config": {
+            "category": "Applications/Databases",
+            "requires": [
+              "libXScrnSaver(x86-64)"
```

Comment:
Though actually this is just an example to document how `requires` works in electron-installer-redhat, we don't configure it in hadron-build, just pass it through. (Lots of hard-won knowledge like the `(x86-64)` suffix behind it too because there is a more confusingly named 32-bit `libxss`, `libXss.so.1` or something like it which I tried first, and also does install but won't fix Compass not starting at all).

Agree with the `Huh?` sentiment though. If we figure out what actually requires it upstream we might be able to remove it too which would be nice but you know, working `.rpm` first, optimise later.